search/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.6.0
+
+### Feature
+* Search service adding suport for AWS Elasticsearch (https://github.com/amundsen-io/amundsen/pull/1141)
```

Comment:
is it correct? We used AWS ES in the beginning.

Reply:
based on https://github.com/amundsen-io/amundsen/pull/1141, the pr is for supporting using IAM role to auth AWS ES.

Reply:
that's correct, but I'm just repeating back the PR titles -- we haven't historically editorialized these